packages/client-core/src/user/UserUISystem.tsx

```diff
@@ -82,6 +84,7 @@ const reactor = () => {
     popupMenuState.menus.merge({
       [UserMenus.Profile]: ProfileMenu,
       [UserMenus.Settings]: SettingMenu,
```

Comment:
If the old settings menu is not being used anymore, we should remove it and replace it with this new one.

Reply:
Oh sorry, I see that this new one is to be used in tailwind routes instead, is that correct?